src/desktop_notifier/sync.py

```diff
@@ -132,3 +132,68 @@ def get_capabilities(self) -> frozenset[Capability]:
         """See :meth:`desktop_notifier.main.DesktopNotifier.get_capabilities`"""
         coro = self._async_api.get_capabilities()
         return self._run_coro_sync(coro)
+
+    @property
+    def on_clicked(self) -> Callable[[str], Any] | None:
+        """
+        A method to call when a notification is clicked
+
+        The method must take the notification identifier as a single argument.
+
+        If the notification itself already specifies an on_clicked handler, it will be
+        used instead of the class-level handler.
+        """
```

Comment:
Could you reference the doc strings from the `DesktopNotifier` properties here instead? This prevents us from needing to keep those texts in sync manually.